docs/source/alfacase/case_description.quick_start.rst

```diff
@@ -3,5 +3,86 @@
 Quick Start
 ===========
 
+In this section, it is shown how to work with a ``CaseDescription`` and how to convert an :program:`ALFAcase`
+from an existent project (exported from ALFAsim) to a ``Description``.
 
-In progress
+CaseDescription
+---------------
+
+The ``CaseDescription`` is a Python class that helps users to create a valid ``.alfacase`` file programmatically, by instantiating a
+:class:`CaseDescription <alfasim_sdk.CaseDescription>` object.
+
+To create a ``CaseDescription`` instance it is necessary to have a Python Interpreter with at least version 3.6 and the
+|sdk| Python package.
+
+
+.. note::
+
+    For more details on how to install Python `check the official docs <https://www.python.org/downloads/>`_.
+    It is recommended that you install the |sdk| using a Python Virtual Environment.
+    For more details, see `Virtual Environments and Packages in Python documentation <https://docs.python.org/3/tutorial/venv.html>`_.
+
+
+From a terminal, and inside a virtual environment, update pip:
+
+>>> python -m pip install -U pip
+
+Install the |sdk| with:
+
+>>> pip install alfasim-sdk
+
+.. note::
+
+    All classes and constants that will be used to configure a project can be accessed through the main module of ``alfasim_sdk``
+
+
+See section :ref:`alfacase-reference-section` for all available classes and options,
+as well as which types and validators are applied at the time of object instantiation.
+
+Knowing the type is important to create instantiate a class properly. For example, if you try to create a
+:class:`NodeDescription <alfasim_sdk.NodeDescription>`, the ``node_type`` attribute
+must be a string of one of the options available on :class:`NodeCellType <alfasim_sdk.NodeCellType>`, otherwise, a
+``ValueError`` is raised.
+
+.. code-block:: python
+
+    >>> import alfasim_sdk
+    >>> alfasim_sdk.NodeDescription(name="Node", node_type=1)
+    [ ... ]
+    ValueError: 'node_type' must be in <enum 'NodeCellType'> (got 1)
+
+    >>> alfasim_sdk.NodeDescription(name='Node', node_type=alfasim_sdk.NodeCellType.Pressure)
+
+
+As already informed at the beginning of the section, ALFAsim accepts only ``.alfacase`` files, the description classes
+are only tools that help the creation and manipulation of a project programmatically in an easier way.
+
+To convert a ``Description`` to alfacase you can use the function ``convert_description_to_alfacase`` and to convert a
```

Comment:
I was actually talking about the functions, they are not consistent with each other, do you think it is worth renaming it?

Reply:
Sure, are your thoughts on this @nicoddemus @prusse-martin 

We will keep

`convert_description_to_alfacase`

And change

`convert_alfacase_to_case` to `convert_alfacase_to_description`

we can keep a function with the old name as backward compatibility at `alfasim_sdk.__init__.py` but IMHO we should remove it immediately =)

Reply:
👍 to `convert_description_to_alfacase` and `convert_alfacase_to_description`.

Agree that we don't need to keep backward compatibility.